common/utils.js

```diff
@@ -486,28 +484,26 @@ const getMultisigProof = async (config, chain, multisigSessionId) => {
 
 const calculateDomainSeparator = (chain, router, network) => keccak256(Buffer.from(`${chain}${router}${network}`));
 
-const getItsEdgeContract = (chainConfig) => {
+const itsEdgeContract = (chainConfig) => {
     const itsEdgeContract =
+        chainConfig.contracts.InterchainTokenService?.objects?.ChannelId || // sui
         chainConfig.contracts.InterchainTokenService?.address ||
-        chainConfig.contracts.ITS?.objects?.ChannelId ||
-        chainConfig.contracts.interchain_token_service?.address;
+        chainConfig.contracts.interchain_token_service?.address; // stellar
 
     if (!itsEdgeContract) {
-        throw new Error(`Missing InterchainTokenService edge contract for chain ${chainConfig.name}`);
+        printError(`Missing InterchainTokenService edge contract for chain: ${chainConfig.name}`);
     }
 
     return itsEdgeContract;
 };
 
-const getItsEdgeChains = (config, excludeChainName) => {
-    return Object.keys(config.chains).filter((chain) => getItsEdgeContract(config.chains[chain]) && chain !== excludeChainName);
-};
-
-const parseTrustedChains = (config, trustedChains, excludeChainName) => {
-    const parsedTrustedChains =
-        trustedChains === 'all' ? getItsEdgeChains(config, excludeChainName) : trustedChains.split(',').map((chain) => chain.trim());
+const itsEdgeChains = (config) =>
+    Object.values(config.chains)
+        .filter(itsEdgeContract)
+        .map((chain) => chain.axelarId);
 
-    return parsedTrustedChains;
+const parseTrustedChains = (config, trustedChains) => {
+    return trustedChains.length === 1 && trustedChains[0] === 'all' ? itsEdgeChains(config) : trustedChains.map((chain) => chain.trim());
```

Comment:
```suggestion
    return trustedChains.length === 1 && trustedChains[0] === 'all' ? itsEdgeChains(config) : trustedChains;
```
the caller should provide a trimmed list (or ideally make the input variadic so it's auto trimmed). weird to do CLI related formatting in a common util